packages/parser/lib/Driver_.re

```diff
@@ -0,0 +1,66 @@
+module Lexer = Css_lexer;
```

Comment:
Any reason why Driver has an underscore?

Reply:
Yes! To avoid conflicts with `ppxlib`'s Driver module because it is opened in `css_to_emotion` in which we also reference `last_buffer`. When the latter is overwritten with a better implementation, I guess we shall be free to remove the underscore, otherwise, I am okay with using another better name :)

Reply:
Can we find another name then?

Reply:
Can we fix the ppxlib driver conflict with Driver, then? and keep "Driver.re" as a name here?

Reply:
No problem!
How is `parse` after we have renamed [Payload.parse](https://github.com/davesnx/styled-ppx/blob/174da50b8c7201c01554b320202f023a77b20f28/packages/ppx/src/payload.re#L39) to `Payload.parse_payload` or `parse_helper`?

Reply:
Yes, done in https://github.com/davesnx/styled-ppx/pull/372/commits/13516ca436eed22a775bbab456a37d91ee352c3e and https://github.com/davesnx/styled-ppx/pull/372/commits/26ec1f6834e2b8e913dc948b863307ffbbc20a00

Reply:
Right, I will rebase on top of the branch containing the commits

Reply:
Done rebasing, meaning https://github.com/davesnx/styled-ppx/pull/372 goes in first instead of  https://github.com/davesnx/styled-ppx/pull/369